src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace GeneralizedHarmonic::BoundaryConditions {
+namespace detail {
+// @{
+/*!
+ * \brief Computes the expression needed to set bnudary conditions on Dt<VPsi>
+ *
+ * \details Computes the correction to the characteristic projected
+ * time-derivatives of the fundamental variables corresponding to \f$u^\psi\f$:
+ *
+ * \f{align}
+ * \partial_t u^{\psi}_{ab} = \partial_t u^{\Psi}_{ab} + v_{\psi} n^i C_{iab}
+ * \f}
+ *
+ * where \f$n^i\f$ is the local unit normal to the external boundary,
+ * and \f$C_{iab} = \partial_i \psi_{ab} - \Phi_{iab}\f$ is the
+ * three-index constraint.
+ */
+template <size_t VolumeDim, typename DataType>
+void set_dt_v_psi_constraint_preserving(
```

Comment:
The use of `v` is a little confusing in the names of the functions and variables -- it sounded like this function was setting a symbol usually denoted `u` in the equation, or do I misunderstand?

Reply:
Agreed, i've changed the help text to use `v` for characteristic fields, and `lambda` for their speeds. This is now consistent with function and variable names, and also with other evolution systems. GH needs to be updated elsewhere to adhere to this variable names, replacing the usage of `u` and `v` for the fields and speeds, respectively. This should be done outside of this PR though.